launch/docker.go

```diff
@@ -21,6 +21,8 @@ var execCommand = exec.Command
 const (
 	ArtifactsDir = "artifacts"
 	LogFile      = "builds.log"
+	ScmHost      = "screwdriver.cd"
+	OrgRepo      = "sd-local/local-build"
```

Comment:
`screwdriver-cd/launcher` was corrected
https://github.com/screwdriver-cd/launcher/pull/331#discussion_r378741057

Reply:
Do we need to add @sugarnaoming 's comment on the code?

Reply:
I see.
fixed